datafusion/core/tests/memory_limit.rs

```diff
@@ -238,7 +238,7 @@ async fn sort_preserving_merge() {
 
 #[tokio::test]
 async fn sort_spill_reservation() {
-    let partition_size = batches_byte_size(&dict_batches());
+    let partition_size = batches_byte_size(&maybe_split_batches(dict_batches(), true));
```

Comment:
What is the rationale for this change?

Reply:
it is needed to match the change in data (which is also split). I tried to make this clearer in 355ef7397.